mssql-plugin/src/e2e-test/java/io.cdap.plugin/common.stepsdesign/TestSetupHooks.java

```diff
@@ -80,4 +83,11 @@ public static void dropTables() throws SQLException, ClassNotFoundException {
                         PluginPropertyUtils.pluginProp("targetTable")});
     }
 
+    @Before(order = 1, value = "@BQ_SINK")
+    public static void setTempTargetBQTable() {
```

Comment:
Write a After Hook as well to delete this target table.

Reply:
resolved